packages/webapp/pages/onboarding.tsx

```diff
@@ -163,6 +161,65 @@ export function OnboardPage(): ReactElement {
     setHasSelectTopics(hasTopics);
   };
 
+  const getContent = (): ReactElement => {
```

Comment:
Wasn't sure why this was "conditionalWrapped" it should have been conditionally rendered as adjusted.

Reply:
I think the conditional wrap came from the requirement of auth options having the custom `title` and `content` as opposed to having just the `AuthOptions` now.